Annotations not being commented in line

Hi @chekalsky - thanks for putting this together. I'm running this on a private repo, but the annotations I get are not being passed to the PR file that is triggering the error.

**Reality**
![image](https://user-images.githubusercontent.com/687789/73344057-8a64e200-42e6-11ea-990a-5d85e4a74fd2.png)

**Expectation**
![image](https://user-images.githubusercontent.com/687789/73343989-699c8c80-42e6-11ea-9712-448e5d6c2e2e.png)

Am I missing something obvious - was hoping for inline annotation like your screenshot.

@chekalsky I have the same issue, not getting any annotations in the file.

PHPCS output:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<checkstyle version="3.5.3">
<file name="/github/workspace/derpcstest.php">
##[warning] <error line="1" column="1" severity="warning" message="A file should declare new symbols (classes, functions, constants, etc.) and cause no other side effects, or it should execute logic with side effects, but should not do both. The first symbol is defined on line 5 and the first side effect is on line 3." source="PSR1.Files.SideEffects.FoundWithSymbols"/>
##[error] <error line="3" column="1" severity="error" message="Expected 1 space(s) after IF keyword; 0 found" source="Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword"/>
##[error] <error line="3" column="11" severity="error" message="Expected 1 space(s) after closing parenthesis; found 0" source="Squiz.ControlStructures.ControlSignature.SpaceAfterCloseParenthesis"/>
##[error] <error line="3" column="12" severity="error" message="Newline required after opening brace" source="Squiz.ControlStructures.ControlSignature.NewlineAfterOpenBrace"/>
##[error] <error line="3" column="26" severity="error" message="Closing brace must be on a line by itself" source="Squiz.WhiteSpace.ScopeClosingBrace.ContentBefore"/>
##[error] <error line="5" column="18" severity="error" message="Opening brace should be on a new line" source="Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine"/>
##[error] <error line="5" column="19" severity="error" message="Closing brace must be on a line by itself" source="Squiz.WhiteSpace.ScopeClosingBrace.ContentBefore"/>
</file>
<file name="/github/workspace/shoarma.php">
##[error] <error line="3" column="17" severity="error" message="Opening brace should be on a new line" source="Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine"/>
##[error] <error line="3" column="18" severity="error" message="Closing brace must be on a line by itself" source="Squiz.WhiteSpace.ScopeClosingBrace.ContentBefore"/>
</file>
</checkstyle>
```

phpcs.xml in repo
```xml
<?xml version="1.0"?>
<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="PHP_CodeSniffer" xsi:noNamespaceSchemaLocation="phpcs.xsd">
    <description>Duracom PSR12</description>
    <rule ref="PSR12"/>
    <file>.</file>
    <exclude-pattern>*/vendor/*</exclude-pattern>
</ruleset>
```

Hope you can help us out :smiley:

--- a/action.yml
+++ b/action.yml
@@ -1,4 +1,4 @@
-name: 'PHP CodeSniffer Check'
+name: 'PHP_CodeSniffer Check'
 description: 'PHPCS checker with annotations out of the box'
 author: 'Ilya Chekalsky'
 inputs:

--- a/problem-matcher.json
+++ b/problem-matcher.json
@@ -5,7 +5,7 @@
       "severity": "error",
       "pattern": [
         {
-          "regexp": "^<file name=\"(.*)\">$",
+          "regexp": "^<file name=\"(?:\\/github\\/workspace\\/)?(.*)\">$",
           "file": 1
         },
         {

